api/index.js

```diff
@@ -13,12 +13,12 @@ module.exports = (state, Config, Logger) => {
   const metaLoader = loaderRegistry.get('metadata')
 
   // load routes
-  const routes = [].concat(
+  const routes = [
```

Comment:
I feel like this change is somehow breaking hapi so I may revert it

Reply:
I wrote it this way because yes, the other way was breaking Hapi. Not 100% sure why!